lib/translator/common.h

```diff
@@ -19,15 +19,31 @@
 #include <type_traits>
 
 #include "absl/types/span.h"
+#include "lib/scsi_defs.h"
+#include "third_party/spdk_defs/nvme_defs.h"
 
 namespace translator {
 
-enum class StatusCode { kSuccess, kInvalidInput, kNoTranslation, kFailure };
+enum class StatusCode {
```

Comment:
nit: might be helpful for future readability to add a comment about the purpose of StatusCode and similarly with ApiStatus. e.g. StatusCode is internal to the translation library